docker/configure_workers_and_start.py

```diff
@@ -488,11 +491,23 @@ def generate_worker_files(
     master_log_config = generate_worker_log_config(environ, "master", data_dir)
     shared_config["log_config"] = master_log_config
 
+    # Find application service registrations
+    appservice_registrations = None
+    appservice_registration_dir = os.environ.get("SYNAPSE_AS_REGISTRATION_DIR")
+    if appservice_registration_dir:
+        # Scan for all YAML files that should be application service registrations.
+        appservice_registrations = [
+            str(reg_path.absolute())
```

Comment:
I can't find any documentation for `absolute()`. Kind of sounds like [`resolve()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve)?

Reply:
Huh, weird
```python
>>> from pathlib import Path
>>> p = Path("/tmp")
>>> p.absolute()
PosixPath('/tmp')
>>> p.__class__
<class 'pathlib.PosixPath'>
>>> help(p.absolute)
```

```
Help on method absolute in module pathlib:

absolute() method of pathlib.PosixPath instance
    Return an absolute version of this path.  This function works
    even if the path doesn't point to anything.
    
    No normalization is done, i.e. all '.' and '..' will be kept along.
    Use resolve() to get the canonical path to a file.
```

Nothing against using `resolve`, this just seemed more intuitive to use (no idea why it's not documented!)

Reply:
It is likely fine, was just confused. 🤷